caddyhttp/httpserver/logger_test.go

```diff
@@ -138,7 +139,9 @@ func TestLoggingToSyslog(t *testing.T) {
 	for i, testCase := range testCases {
 
 		ch := make(chan format.LogParts, 256)
-		server, err := bootServer(testCase.Output, ch)
+		if server, err := bootServer(testCase.Output, ch); err != nil {
```

Comment:
`server` and `err` here are scoped within the `if` block. The rest of the func can't see them, which causes the tests and compilation to fail with this:
```
caddyhttp/httpserver/logger_test.go:145:9: undefined: server
caddyhttp/httpserver/logger_test.go:147:6: undefined: err
caddyhttp/httpserver/logger_test.go:148:82: undefined: err
```

Reply:
I see, my bad...
Also, I'm on a Windows machine so I cant build this Darwin file nor do I see lints

Reply:
Try running `go test ./...` from the project's root directory.